pkg/reconciler/revision/config/store.go

```diff
@@ -35,13 +35,12 @@ type cfgKey struct{}
 // +k8s:deepcopy-gen=false
 // Config contains the configmaps requires for revision reconciliation.
 type Config struct {
-	Defaults      *config.Defaults
+	*config.Config
```

Comment:
Do you need to do anything to set this up in NewStore?

Reply:
it's setup in `Load()`